### DISCUSSION TITLE
Connection Error on Code for IBM-i in VS Code as "Check your credentials (All configured authentication methods failed)."

### DISCUSSION THREAD

**Discussion by Nobutaka-Saegusa** [codefori/vscode-ibmi#2427] [Q&A]
I got the following error when I tried to connect to IBM-i through Code for IBM-i in VS code. 

Not connected to xx.xxx.xxxx! Check your credentials (All configured authentication methods failed).

I changed the authority of the IBM-i user profile I use as having "*ALLOBJ" and tried again, the result was the same.

In case this was caused by the wrong/insuficient  authority code settings on user profile, what combination of it is required to be set?
Or how I can change the authentication methods to be connected normally?
The sshd job QP0ZSPWP on IBM-i is already raised up.
It was connected normally when I  ping to it  from terminal.

**Comment by sebjulliand** [comment]
Hi @Nobutaka-Saegusa ,
that message means you provided the wrong credentials (usually wrong password or private key) or there are some special characters in the password that prevents it from being used through SSH.

First of all, check you can connect through SSH, using Putty under Windows or the ssh client under *nix OSes.
If you can't connect through SSH and you double checked your credentials, then the most probable issue will be the special characters in your password. Which means you'll have to:
- Use standard characters in your password (0-9, A-Z)
- Or use private key authentication

But first, try SSH outside of VS Code and see how it goes.

**Reply by KVRPavanShanmukh** [reply]
Thanks a lot sir, make your response visible to everyone whomever face this issue.
For me on reading this discussion : Just a change in the password, changed everything.
